content/external-tools/kicost.adoc

```diff
@@ -1,9 +1,9 @@
 +++
 title = "KiCost"
 toolimage = "/img/external-tools/kicost.png"
-tooldeveloper = "Dave Vandenbout"
+tooldeveloper = "Dave Vandenbout (author), Hildo Guillardi Jr. (main collaborator)"
 toolurl = "https://github.com/xesscorp/KiCost"
 +++
 
-KiCost is intended to be run as a script for generating part-cost spreadsheets for circuit boards developed with KiCad.
-
+KiCost is intended to run as an script for generating part-cost spreadsheets for circuit boards developed with KiCad scrapping web distributors to get the parts cost. Nowdays KiCost also support other EDAs BoM files (useful for yours older projects and handle made list on CSV), also have a graphical interface for the users (type `kicost` on terminal to access it).
```

Comment:
Sorry, but your grammar is wrong :/ "an" is only used before vowels, s is not a vowel. 

Better english:

"KiCost is intended to be run as a script for generating part-cost spreadsheets for circuit boards developed with KiCad. It works by scraping part distributor websites to get part pricing. Additionally, KiCost supports other bill of material files from other EDAs which is useful for older projects and it can even handle hand-made CSV files. KiCost also comes with a graphical user interface in addition to command line."